src/dmd/globals.d

```diff
@@ -233,6 +233,11 @@ extern (C++) struct Param
     uint versionlevel;                  // version level
     Array!(const(char)*)* versionids;   // version identifiers
 
+    debug(LOG)
+    {
+        Array!(const(char)*)* calls; //Module or Function Call
```

Comment:
Probably easier to unconditionally declare this, given this also needs to be reflected in the headers.

Also a more descriptive comment would be useful. What Module or Function Call. why?

Reply:
Done.